docs/source/tutorials/object_detection.ipynb

```diff
@@ -0,0 +1,533 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2e9ffd6f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "IMAGE_PATH = './example_images/'  # path to raw image files downloaded above\n",
+    "predictions = pickle.load(open(\"predictions.pkl\", \"rb\"))\n",
+    "labels = pickle.load(open(\"labels.pkl\", \"rb\"))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "35d49e5d",
+   "metadata": {},
+   "source": [
+    "In object detection datasets, each given label is a made up of bounding box coordinates and a class label. A model prediction is also made up of a bounding box and class label as well as the model confidence in the prediction (i.e. probability the prediction is correct). For both `get_label_quality_scores` and `find_label_issues`, cleanlab requires a list of given labels and a list of model predictions for all examples.\n",
+    "\n",
+    "Here’s what an example looks like in our dataset. We can visualize the bounding boxes and labels using `cleanlab.object_detection.summary.visualize` function. Here the given labels are in red and the predicted in blue."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "56705562",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "image_to_visualize = 8  # change this to view other images\n",
+    "image_path = IMAGE_PATH + labels[image_to_visualize]['seg_map']\n",
+    "visualize(image_path, label=labels[image_to_visualize], prediction=predictions[image_to_visualize], overlay=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ff36d97f",
+   "metadata": {},
+   "source": [
+    "The expected format of these `labels` and `predictions` is modeled after what common models like mmdet and detectron2 expect. The 5 possible class labels in our dataset are car, chair, cup, person, traffic light. These classes are zero-indexed and represented with an integer 0,1,...,4. \n",
+    "\n",
+    "The 5 possible class labels in our dataset are: car, chair, cup, person, traffic light. These classes are zero-indexed and represented with an integer 0,1,...,4.\n",
+    "\n",
+    "`labels` is a list where for the i-th image in our dataset, `labels[i]` is a dictionary containing: key `labels` -- a list of class labels for each bounding box in this image and key `bboxes` -- a numpy array of the bounding boxes' coordinates. Each bounding box in `labels[i]['bboxes']` is in the format ``[x1,y1,x2,y2]`` where `(x1,y1)` corresponds to the bottom-left corner of the box and `(x2,y2)` the top-right.  \n",
+    "\n",
+    "Let's see what `labels[i]` looks like for our previous example image:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b08144d7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "labels[example_idx]"
```

Comment:
`example_idx` here is not defined, same with the code block below for `predictions[example_idx]`